servers/rest-services/src/main/java/org/projectnessie/services/rest/RestV2ConfigResource.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.rest;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.Path;
+import org.projectnessie.services.config.ServerConfig;
+
+/** REST endpoint to retrieve server settings. */
+@RequestScoped
+@Path("v2beta/config")
+public class RestV2ConfigResource extends RestConfigResource {
+
```

Comment:
Would be nicer IMO to not extend the V1 REST class but implement the HttpConfigApi interface from the v2 package.

Reply:
reworked